src/yunohost/app.py

```diff
@@ -1415,6 +1415,9 @@ def _extract_app_from_file(path, remove=False):
             manifest['lastUpdate'] = int(time.time())
     except IOError:
         raise MoulinetteError(errno.EIO, m18n.n('app_install_files_invalid'))
+    except ValueError as e:
+        raise MoulinetteError(errno.EINVAL,
+                              m18n.n('app_manifest_invalid', error=e.strerror))
```

Comment:
The 'app_manifest_invalid' string does not expect to have a 'error' arg. Doesn't that trigger and error ?